DemoApp-Java/app/src/main/res/layout/activity_layout_editor_banner_ad.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".ads.LayoutEditorBannerAdActivity">
+
+    <com.applovin.mediation.ads.MaxAdView xmlns:maxads="http://schemas.applovin.com/android/1.0"
```

Comment:
I think `xmlns:maxads="http://schemas.applovin.com/android/1.0"` should be moved into the parent `ConstraintLayout` definition above. Usually all the XML namespace definitions are set in the root view of the layout.